front-end/src/components/gallery/Gallery.js

```diff
@@ -154,7 +224,7 @@ const Gallery = () => {
                             <PagButton onClick={(() => setCurrPage(index + 1))} currPage={currPage} index={index + 1} key={index}>{index + 1}</PagButton>
                         ))}
                     </div>
-                    {products.results.length >= 12 && (<Button onClick={(() => setCurrPage(currPage + 1))}>NEXT</Button>)}
```

Comment:
Extract `12` into constant variable in a file with constant like `itemsPerPage` so you easily change it on one place in the future if needed.